docs/demos.md

```diff
@@ -3,8 +3,14 @@ title: Where can I see the web and mobile demos?
 sidebar_label: Web and Mobile demos
 ---
 
-_This page is a stub._
+## Web
+* Deployed version: **https://kiwicom-margarita.netlify.com**
```

Comment:
Maybe I would put the URLs as links to the same URL so they are clickable

Reply:
Docusaurus can handle this and they are already clickable.

Reply:
Magic * ___ *